src/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Please make sure that in the constructor method, the rental is adding itself to the @book achieving the 'belongs-to' association like :point_right: `book.rentals << self` and the rental is adding itself to the @person the 'belongs-to' association like :point_right:  `person.rentals << self` :100: :pray:

Reply:
Fixed.